docs/HowTo/Interact/APIs/Using-JSON-RPC-API.md

```diff
@@ -120,5 +120,5 @@ The block parameter can have the following values:
 * `pending` : `tag` - Last block mined plus pending transactions. Use only with [eth_getTransactionCount](../../../Reference/API-Methods.md#eth_gettransactioncount).  
 
 !!! note
-    If [synchronizing in FAST mode](../../../Reference/CLI/CLI-Syntax.md#fast-sync-options), most historical 
-    world state data is unavailable.  Any methods attempting to access unavailable world state data return `null`.
\ No newline at end of file
+    If [synchronizing in FAST mode](../../../Reference/CLI/CLI-Syntax.md#sync-mode), most historical 
```

Comment:
@MadelineMurray As part of removing the sync options section, I had to update this link. However, the link isn't very helpful. I think there's a small information gap that's beyond the scope of this PR. Namely, some kind of explanation of full mode and fast mode could be useful in the docs.

Reply:
This makes sense - could you raise a JI to add this content.

Reply:
Done.